techtv2ovs/tasks.py

```diff
@@ -0,0 +1,90 @@
+""" Celery tasks for techtv2ovs """
+
+from os.path import splitext
+
+import boto3
+from celery import shared_task
+from celery.utils.log import get_task_logger
+from django.conf import settings
+
+from techtv2ovs.constants import TTV_VIDEO_BUCKET, ImportStatus
+from techtv2ovs.models import TechTVVideo
+from ui.models import VideoFile
+
+log = get_task_logger(__name__)
+
+# pylint: disable=unused-argument,broad-except
+
+
+def parse_encoding(filename):
+    """
+    Get the correct encoding base on the video file's name
+
+    Args:
+        filename (str): The video file name
+
+    Returns:
+        str: The encoding for the videofile.
+
+    """
+    basename = splitext(filename)[0].split('/')[-1]
+    return basename.upper() if basename == 'hd' else basename
```

Comment:
The encoded video filenames are always the same ('basic.mp4', 'medium.mp4', 'hd.mp4', etc) but I can modify this just in case there's an exception to the rule somewhere.